temp/legacy-code/src/main/java/com/ivy/legacy/ui/component/transaction/TransactionCard.kt

```diff
@@ -346,9 +343,15 @@ private fun TransactionHeaderRow(
                 accounts = accounts
             )
 
+            val accountBackgroundColor = if (shouldShowAccountSpecificColorInTransactions) {
+                account?.color?.toComposeColor()
```

Comment:
Add the `?: UI.colors.pure` here to simplify the code a bit

Reply:
Done